sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -104,3 +106,25 @@ def test_check_estimators_unfitted():
     # check that CorrectNotFittedError inherit from either ValueError
     # or AttributeError
     check_estimators_unfitted("estimator", CorrectNotFittedErrorClassifier)
+
+
+def test_common():
+    try:
+        import pandas as pd
+    except ImportError:
+        raise SkipTest("Pandas not found")
```

Comment:
No, we're doing something like this elsewhere, e.g. `test_cross_val_score_pandas`